lib/RequestHandler.php

```diff
@@ -13,14 +13,16 @@ private static function createUrl(string $resource): string {
         return sprintf("%s/v%d/%s", Settings::$baseUrl, Settings::$apiVersion, ltrim($resource, "/"));
     }
 
-    private static function createHeaders(bool $withBody): array {
+    private static function createHeaders(bool $withBody, bool $isDownload = false): array {
         $headers = [
             "User-Agent" => sprintf("Trunkrs SDK/PHP/v%s", Settings::$sdkVersion),
             "X-API-ClientId" => Settings::$clientId,
             "X-API-ClientSecret" => Settings::$clientSecret,
-            "Accept" => "application/json; charset=utf-8",
         ];
 
+        if (!$isDownload) {
+            $headers["Accept"] = "application/json; charset=utf-8";
+        }
         if ($withBody) {
             $headers["Content-Type"] = "application/json; charset=utf-8";
```

Comment:
If its a download should we not also provide the content-type?

Reply:
This is the `Content-Type` for the request body, when a `PUT` request is done in download mode and a body has been provided this header will be added as well. 🙂

Reply:
a ok, would expect doing a download request though we would accept also contents that differ from json? such as pdf for instance?